mkosi

```diff
@@ -4662,6 +4671,7 @@ def print_summary(args: CommandLineArguments) -> None:
     sys.stderr.write("        FS Compression: " + yes_no(args.compress) + detail + "\n")
 
     sys.stderr.write("        XZ Compression: " + yes_no(args.xz) + "\n")
+    sys.stderr.write("        ZSTD Compression: " + yes_no(args.zstd) + "\n")
```

Comment:
I'm not sure, but I think the colons are supposed to line up.

Reply:
fixed